packages/babel-preset-razzle/index.js

```diff
@@ -52,9 +50,20 @@ if (env === 'test') {
       require.resolve('babel-plugin-transform-es2015-modules-commonjs'),
       { loose: true },
     ],
+    // Add support for async/await
+    [
+      require.resolve('babel-plugin-transform-runtime'),
+      {polyfill: false},
+    ],
   ]);
 }
 
+if (env === 'production' || env === 'development') {
```

Comment:
what's the point of this statement, exactly? do any environments besides production/development exist? 😬

Reply:
test? it could be simplified to
```
if (env !== 'test')
```

I just found easier to read, that specific rule concerns those two environments

Reply:
Ah. Fair.